govcd/nsxt_alb_virtual_service_test.go

```diff
@@ -78,6 +85,8 @@ func (vcd *TestVCD) Test_AlbVirtualService(check *C) {
 	// cleanup Org user
 	err = orgUser.Delete(true)
 	check.Assert(err, IsNil)
+
+	failedAlbCleanup = false
```

Comment:
If the test got here without failures - there is no need to run deferred cleanup